packages/app-web/README.md

```diff
@@ -5,4 +5,4 @@ Core web app built on React and Redux Toolkit
 
 ## Development setup
 
-Follow the instructions in the 'Development setup' section in the project root.
+Follow [the instructions](https://github.com/wierkstudio/ciphereditor#development-setup) in the 'Development setup' section in the project root.
```

Comment:
We should use relative paths here, as the repository may be viewed outside of GitHub.

```suggestion
Follow [the instructions](../../README.md#development-setup) in the 'Development setup' section in the project root.
```

Reply:
Very fair - I'm currently somewhat inexperienced right now, so this should be good. :)